gulpfile.js

```diff
@@ -160,8 +164,8 @@ gulp.task('webserver', () => {
 
 gulp.task('watch', () => {
 	gulp.watch('src/pug/**/*.pug', gulp.series('pug'));
-	gulp.watch('src/styl/**/*.styl', gulp.series('clean:styl', 'styl'));
-	gulp.watch('src/js/*.js', gulp.series('clean:js', 'js'));
+	gulp.watch('src/styl/**/*.styl', gulp.series('styl'));
```

Comment:
Disabling these clean tasks is not required.
Give me your point about it!

Reply:
@vikaspotluri123 wdyt?

Reply:
It's not because watch should only run in dev mode -> cachebusting is disabled